tests/foreman/cli/test_domain.py

```diff
@@ -130,54 +130,65 @@ class DomainTestCase(CLITestCase):
     """Domain CLI tests"""
 
     @tier1
-    def test_positive_create_with_name_description(self):
-        """Create domain with valid name and description
+    @upgrade
+    def test_positive_create_update_delete_domain(self):
+        """Create domain, update and delete domain and set parameters
 
         :id: 018740bf-1551-4162-b88e-4d4905af097b
 
-        :expectedresults: Domain successfully created
+        :expectedresults: Domain successfully created, updated and deleted
 
 
         :CaseImportance: Critical
         """
-        for options in valid_create_params():
-            with self.subTest(options):
-                domain = make_domain(options)
-                self.assertEqual(domain['name'], options['name'])
-                self.assertEqual(
-                    domain['description'], options['description'])
-
-    @tier1
-    def test_positive_create_with_loc(self):
-        """Check if domain with location can be created
-
-        :id: 033cc37d-0189-4b88-94cf-97a96839197a
-
-        :expectedresults: Domain is created and has new location assigned
-
-
-        :CaseImportance: Medium
-        """
+        options = valid_create_params()[0]
         location = make_location()
-        domain = make_domain({'location-ids': location['id']})
+        org = make_org()
+        domain = make_domain({
+            u'name': options['name'],
+            u'description': options['description'],
+            u'location-ids': location['id'],
+            u'organization-ids': org['id']
+            })
+        self.assertEqual(domain['name'], options['name'])
+        self.assertEqual(domain['description'], options['description'])
         self.assertIn(location['name'], domain['locations'])
+        self.assertIn(org['name'], domain['organizations'])
 
-    @tier1
-    def test_positive_create_with_org(self):
-        """Check if domain with organization can be created
-
-        :id: f4dfef1b-9b2a-49b8-ade5-031da29e7f6a
-
-        :expectedresults: Domain is created and has new organization assigned
+        # set parameter
+        parameter_options = valid_set_params()[0]
+        parameter_options['domain-id'] = domain['id']
+        Domain.set_parameter(parameter_options)
+        domain = Domain.info({'id': domain['id']})
+        parameter = {
+            # Satellite applies lower to parameter's name
+            parameter_options['name'].lower(): parameter_options['value'],
+        }
+        self.assertDictEqual(parameter, domain['parameters'])
+
+        # update domain
+        options = valid_update_params()[0]
+        Domain.update(dict(options, id=domain['id']))
+        # check - domain updated
```

Comment:
I think this comment not requires,  as it's expected to see the same to check in assert .